spec/classes/foreman_proxy__plugin__pulp_spec.rb

```diff
@@ -39,5 +39,16 @@
           :group   => 'example'
         })
     end
+
+    it 'should generate correct pulp.yml' do
+      varify_exact_contents(catalogue, "#{etc_dir}/foreman-proxy/settings.d/pulp.yml", [
```

Comment:
This should be verify instead of varify. Other than that you need to set the etc_dir variable.